en/docs/deploy/self-hosted/docker.md

```
@@ -0,0 +1,133 @@
+```bash
+export DOCKER_DEFAULT_PLATFORM=linux/amd64
+```
+
+This setting applies only to the current terminal session.
+:::
+
+## How Code to Cloud works
+
+When you build a Ballerina project with a cloud target, the compiler extension generates deployment artifacts alongside the executable JAR. The artifacts land in the `target/` directory:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)